src/main/credentialServer/httpCredentialHelperServer.ts

```diff
@@ -150,6 +157,26 @@ export class HttpCredentialHelperServer {
     }
   }
 
+  protected generateVtunnelConfig() {
```

Comment:
This seems _very_ weird to be here: instead, we should be doing this somewhere in `wsl.ts` (or a new file), and just get the data from this instead.  `vtunnel` is much larger than just the credential helper server.